src/store/actions.ts

```diff
@@ -644,10 +715,12 @@ export const setStaffPayment: ActionIF = ({ commit }, staffPayment: StaffPayment
 export const setFolioNumber: ActionIF = ({ commit }, folioNumber: string): void => {
   commit('mutateFolioNumber', folioNumber)
 }
-/** Name Check actions
+
+/**
+ * Name Check actions
  * FUTURE: move these into a factory if converting to composition api
  */
-export const getMatchesExact = async (
```

Comment:
There are only used locally (and this isn't a class so I couldn't make them private), so I just stopped exporting these.